documentation/docs/getting-started/decentralized_identifiers/resolve.md

```diff
@@ -5,5 +5,47 @@ description: Explain how resolving works including arguments.
 image: /img/Identity_icon.png
 keywords:
 - Resolve
+- DID
 ---
-TODO: Explain how resolving works including arguments.
\ No newline at end of file
+
+In the previous section we created a decentralized identity and published the associated DID Document to the Tangle; in this section we'll show you how to look up a DID document.
```

Comment:
I try (and often fail) to avoid writing these things as they would be spoken. "WE did xyz, next WE'll show YOU". Not even sure if that is a good idea. Food for thought ;)

Reply:
🙈 I always default to passive voice when writing instead of direct active voice haha! Re-worded a little bit here to not talk about the humans.